docs/framework/interop/how-to-map-hresults-and-exceptions.md

```diff
@@ -49,36 +49,29 @@ CMyClass::MethodThatThrows
 }  
 ```  
   
- The following table provides the complete mapping from each HRESULT to its comparable exception class in the .NET Framework.  
+ The following table provides the common mappings from HRESULT to its comparable exception class in the .NET. HRESULT values without explicit mappings are mapped to **COMException**. The complete up-to-date mapping can be found in [dotnet/runtime repo](https://github.com/dotnet/runtime/blob/master/src/coreclr/vm/rexcep.h). 
```

Comment:
I am not able to figure out what the markdown lint is complaining about.

Reply:
It's mad that you have a space at the end of the line, you should have 0. Most articles that were automatically ported added 2 spaces to all lines, which generally get cleaned up when the article gets a full refresh. But 0 is what you should use.

Reply:
> It's mad that you have a space at the end of the line, you should have 0. 

I have removed the trailing whitespaces from the whole file. Use https://github.com/dotnet/docs/pull/22487/files?w=1 to see the real delta without the whipespace only changes.